app/views/events/_form.html.erb

```diff
@@ -44,6 +49,34 @@
     <%=f.error_span(:active) %>
   </div>
   <div class="form-group">
+    <%= f.label :date_ranges, :class => 'control-label col-lg-2' %>
+    <div class="col-lg-10" id="event-date-pickers">
+      <% def date_picker_template(startDate = Date.today, endDate = Date.today)
+          '<div class="event-date-picker"><div class="event-date-picker-start">' + 
+          select_date(startDate, {prefix: 'date_ranges[start_date][]'}).gsub("\n", "") +
+          '</div> bis <div class="event-date-picker-end">' +
+          select_date(endDate, {prefix: 'date_ranges[end_date][]'}).gsub("\n", "") +
+          '</div></div>'
```

Comment:
This hunk might just benefit from string interpolation, date calculations should be extracted